docs/commands/deploy.md

```diff
@@ -98,6 +98,7 @@ netlify deploy
 - `timeout` (*string*) - Timeout to wait for deployment to finish
 - `trigger` (*boolean*) - Trigger a new build of your site on Netlify without uploading local files
 - `build` (*boolean*) - Run build command before deploying
+- `bundle` (*boolean*) - Ignore functions manifest file, forcing functions to always be bundled as part of the deployment
```

Comment:
I'm not too sure about the name of this flag. I think it should be clear that it overrides the default behaviour, so perhaps it should be something like `--force-bundle` or `--ignore-cache`? 🤔

Reply:
We might want to see it more from the perspective of the users. For example, the users would not know what the manifest file is. From their perspective, they might only care about whether their Functions are uploaded or not.

I might be missing some details, but what do you think about something like: `--skip-functions-cache` with a description along the lines of "Deploy the Functions even if they have been already deployed". I think this is most likely inaccurate and I might be not understanding the full intent behind this flag, so please feel free to correct me.

Reply:
Agreed 100%! I've changed it to `--skip-functions-cache` and removed any references to "manifest file" from the user-facing messages.